app/views/group/index.html.erb

```diff
@@ -0,0 +1,22 @@
+  <div class="group-container">
+    <h1 class="group_title">Categories</h1>
+    <% @groups.each do |group| %>
+    <div class="group">
+    <div class="amount-name">
+    <h2><%= link_to group.name, group_path(group), class: 'group_link'%></h2>
+          <p class="group_amount">
+            $<%= group.items.sum(:amount).to_f %>
+          </p>
+    </div>
+      <div class="group_item">
+        <p class="group_p"><%= "#{group.icon}" %></p>
+          <p class="group_date"><%= group.created_at.strftime("%B %d, %Y") %></p>
+          </div>
+    </div>
+    <% end %>
+  </div>
+  <div class="add_new_container">
+<div class="add_new">
```

Comment:
- This button should be the same color provided in the original design which is `#5fb523`. 
### Your implementation
![image](https://github.com/david-oganization/rails-capstone-project/assets/45460192/f99dd002-da78-406f-8d5d-6da473cbd9de)